runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -217,33 +219,52 @@ public void processFlash(final Graphics2D graphics)
 			return;
 		}
 
-		if (Instant.now().minusMillis(MINIMUM_FLASH_DURATION_MILLIS).isAfter(flashStart))
+		final int flash_notification_duration_config_val = runeLiteConfig.flashNotificationDuration();
+		final int flash_duration_millis;
```

Comment:
Our code convention is to use camelCase for variables. These vars can be much more brief anyway.
```suggestion
		final int configFlashDurationMillis = runeLiteConfig.flashNotificationDuration();
		final int flashDurationMillis;
```